src/gui/qgsmaptoolcapture.cpp

```diff
@@ -378,8 +383,22 @@ void QgsMapToolCapture::cadCanvasMoveEvent( QgsMapMouseEvent *e )
   {
     bool hasTrace = false;
 
+    if ( mStreamingEnabled )
+    {
+      if ( ! mCaptureCurve.isEmpty() )
+      {
+        QgsSettings settings;
+        const int tolerance = settings.value( QStringLiteral( "/qgis/digitizing/stream_tolerance" ), 2 ).toInt();
```

Comment:
I don't think it'd ever be a problem, but I think that'd be better only called once per 'stream' segment, i.e. on pushing the button down, instead of every time the mouse moves.

Reply:
Done